---
title: "Regina H Reynolds's CV"
author: Regina H Reynolds
date: "2025-06-22"
params:
  pdf_mode:
    value: true
output:
  pagedown::html_resume:
    css: ['../css/dd_cv.css', 'resume']
    self_contained: true
    includes:
      after_body: ../css/footer.html
---





<style>
:root{
  --decorator-outer-offset-left: -6.5px;
}
</style>


Aside
================================================================================




View this CV online with links at https://rhreynolds.github.io/cv

Contact {#contact}
--------------------------------------------------------------------------------

- <i class='fa fa-envelope'></i> rhreynolds@hotmail.co.uk
- <i class='fa fa-github'></i> [GitHub](http://github.com/RHReynolds)
- <i class='fa fa-linkedin'></i> [LinkedIn](https://www.linkedin.com/in/regina-reynolds-3649b51a/)
- <i class='fa fa-book'></i> [ResearchGate](https://www.researchgate.net/profile/Regina-Reynolds-6)

Languages {#languages}
--------------------------------------------------------------------------------

- <i class='fa-brands fa-r-project'></i> R
- <i class='fa-solid fa-code'></i> Bash
- <i class='fa-brands fa-python'></i> Python

Tools {#tools}
--------------------------------------------------------------------------------

- <i class='fa-brands fa-git-alt'></i> Git/GitHub/GitLab
- <i class='fa-solid fa-arrows-split-up-and-left'></i> Nextflow
- <i class='fa-brands fa-docker'></i> docker

Disclaimer {#disclaimer}
--------------------------------------------------------------------------------

The source code is available [GitHub](https://github.com/RHReynolds/cv).

Last updated on 2025-06-22.



Main
================================================================================

Regina H Reynolds {#title}
--------------------------------------------------------------------------------

Molecular biologist turned bioinformatician, with experience translating large-scale omics data into actionable insights for drug discovery. I’ve led cross-functional analyses across transcriptomics, genomics, and functional screens, collaborating closely across teams to build reproducible pipelines that support impactful R&D decisions. I’m driven by a commitment to scientific rigour, integrity, and transparency — and by a broader desire to make a meaningful contribution to the world through thoughtful, data-driven science.


Work experience {data-icon=suitcase}
--------------------------------------------------------------------------------

### Lead Computational Biologist

CoSyne Therapeutics

London, UK

Present - 2022

- Provide scientific and strategic leadership as lead of a 5-person computational biology team, translating high-level objectives into clear milestones and actionable plans, managing tight deadlines and resources, and setting research priorities in collaboration with team members, experimental, engineering, and AI teams, and senior leadership.
- Drive company-wide 'omics initiatives spanning  whole-genome sequencing, transcriptomics and CRISPRi screens to support target discovery in glioblastoma multiforme.
- Establish scalable, robust, and reproducible computational workflows through adoption of tools such as Nextflow and Seqera Cloud for workflow orchestration, Docker for containerisation, R targets for reproducible exploratory data analysis, and Cruft templates for project scaffolding.
- Contribute to shaping company culture through line management; growing the computational biology team from 2 to 5 people; active involvement in recruitment across multiple teams; and contributing to the development of CoSyne’s personal development framework.



### Honorary Senior Research Fellow

University College London

London, UK

2025 - 2022

- Acted as secondary supervisor to a PhD student and served on the thesis committee for another. Additionally contributed to analyses and provided input on papers across multiple research projects.



### Research Fellow

University College London

London, UK

2022 - 2021

- Lead analyst involved in processing and analysing transcriptomic data to identify molecular signatures of Parkinson's disease progression. Worked within a large, multi-PI team and coordinated with researchers to align analytical approaches. Work performed using R, Nextflow, and Docker.
- Co-lead of Code and Pipeline Alignment Working Group in the Aligning Sciences Across Parkinson's<sup>1</sup> initiative, facilitating the standardisation of data processing pipelines and coding practices across 10 international research teams, resulting in actionable recommendations to enhance data harmonisation and enable meta-analysis of post-mortem brain tissue datasets.
- Published 1 co-first author research article.



### Research Assistant

University of Copenhagen

Copenhagen, Denmark

2016 - 2016

- Led project exploring the interactions between miR-34a, Sirt1 and p53 in a Huntington’s disease mouse model, serving as lead experimentalist and analyst; work culminated in a first author publication<sup>2</sup>.
- In addition to these roles, facilitated collaboration with another research group and coordinated experiments with lab technicians, students, and researchers to support project goals.


Education {data-icon=graduation-cap data-concise=true}
--------------------------------------------------------------------------------

### PhD, Bioinformatics

University College London

London, UK

2021 - 2016

- Thesis: Exploring the importance of cell-type-specific gene expression regulation and splicing in Parkinson's disease<sup>3</sup>.
- Integrated GWAS summary statistics with bulk/single-cell transcriptomic, eQTL, and chromatin accessibility data to identify cell-type-specific regulatory mechanisms in Parkinson’s disease. Methods used included partitioned heritability, COLOC, and MAGMA.
- Applied transcriptomic methods, including cell-type deconvolution, differential splicing analysis, and RNA-binding protein motif analysis, to investigate splicing alterations and their relevance to Lewy body diseases.
- Published 3 first/co-first author research articles and 1 first author review. Successfully secured £10,000 from Signe og Peter Gregersens Mindefond to undertake transcriptional profiling of Parkinson’s disease brain tissue.



### MSc, Molecular Biomedicine

University of Copenhagen

Copenhagen, Denmark

2016 - 2014

- Thesis: Changes in the miR-34a-SIRT1 axis in Huntington’s disease
- Grade: A (92.5%)



### BSc, Molecular Biomedicine

University of Copenhagen

Copenhagen, Denmark

2013 - 2010

- Thesis: Pro-apoptotic factors in Huntington's disease: a study in the R6/2 transgenic mouse model
- Grade: A (96.7%)




<br>


Teaching experience {data-icon=chalkboard-teacher}
--------------------------------------------------------------------------------

### Subsidiary PhD Supervisor

University College London

London, UK

2025 - 2021

- Secondary supervisor to Dr. Aaron Wagen. Provided support around transcriptomic analyses, bioinformatics workflows and statistical genetics.



### R fundamentals with Clinician Coders<sup>4</sup>

University College London

London, UK

2022 - 2019

- Developed materials<sup>5</sup> and led workshops teaching basic R and tidy data principles to clinical academics.



### Omics Techniques

King's College London

London, UK

2019 - 2017

- Lectured graduate level students on the principles of genome-wide association studies and led a workshop on how/why to use the Genotype-Tissue Expression portal.


Voluntary Work {data-icon=hands-helping}
--------------------------------------------------------------------------------

### Peer Reviewer

N/A

London, UK

Present - 2018

- Reviewer<sup>6</sup> for several scientific journals.



### Mentor

[Social Mobility Foundation](https://www.socialmobility.org.uk/)

London, UK

2022 - 2017

- Mentored 4 A-level students looking to work in the field of biomedical research.


Key publications {data-icon=book}
--------------------------------------------------------------------------------

::: aside
A full list of publications is available online at 
https://rhreynolds.github.io/cv
:::

### Local genetic correlations exist among neurodegenerative and neuropsychiatric diseases<sup>7</sup>

NPJ Parkinson's disease

N/A

2023

- **Reynolds, RH**, Wagen, AZ, Lona-Durazo, F, Scholz, SW, Shoai, M, Hardy, J, Gagliano Taliun, SA, Ryten, M
- **Role**: Co-first author, lead analyst and corresponding author.
- Analysis<sup>8</sup> of local genetic correlations between neurodegenerative and neuropsychiatric disorders, with the aim of identifying genomic regions and genes that may drive pleiotropy.



### Cross-platform transcriptional profiling identifies common and distinct molecular pathologies in Lewy body disorders<sup>9</sup>

Acta Neuropathologica

N/A

2021

- Feleke, R, **Reynolds, RH**, Smith, A, Tilley, B, Gagliano Taliun, SA, Hardy, J, Matthews, PM, Gentleman, S, Owen , D, Johnson, MR, Srivastava, P, Ryten, M
- **Role**: Co-first author and analyst.
- Transcriptomic analysis<sup>10</sup> of cell-type-specific changes in the Lewy body diseases.



### Informing disease modelling with brain-relevant functional genomic annotations<sup>11</sup>

Brain

N/A

2019

- **Reynolds, RH**, Hardy, J, Ryten, M, Gagliano Taliun, SA
- **Role**: First author.
- Review of conceptual advances in the generation of brain-relevant functional genomic annotations and among tools that allow integration of these annotations with genome-wide association summary statistics.



### Moving beyond neurons: the **Role** of cell type-specific gene regulation in Parkinson’s disease heritability<sup>12</sup>

NPJ Parkinson's disease

N/A

2019

- **Reynolds, RH**, Botía, JA, Nalls, MA, International Parkinson’s Disease Genomic Consortium (IPDGC), System Genomics of Parkinson’s Disease (SGPD), Hardy, J, Gagliano Taliun, SA, Ryten, M
- **Role**: First author and lead analyst.
- Analysis of Parkinson's disease common variation, with the aim of identifying cell types and pathways of importance to disease risk.


Conferences {data-icon=comments}
--------------------------------------------------------------------------------

### AD/PD, Alzheimer's & Parkinson's Diseases Conference

N/A

Hybrid event

2022

- **Talk**: Identifying genetic correlations among neurodegenerative and neuropsychiatric diseases



### Genomics of Brain Disorders

N/A

Virtual event

2021

- **Talk**: Dysregulation of splicing in human brain from individuals with Lewy body disease informs disease mechanisms



### International Parkinson’s Disease Genomics Consortium

N/A

London, UK

2019

- **Talk**: Pairing bulk and single-nuclear RNA-seq to identify dementia-related pathways in PD



### AD/PD, Alzheimer's & Parkinson's Diseases Conference

N/A

Lisbon, Portugal

2019

- **Talk**: Mapping Parkinson's disease heritability to specific brain cell types
- Received mention in a blog post on Alzforum<sup>13</sup>.



### International Parkinson’s Disease Genomics Consortium

N/A

Reykjavik, Iceland

2018

- **Talk**: Moving beyond neurons: exploring the importance of cell type-specific gene expression in Parkinson’s disease



Links {data-icon=link}
--------------------------------------------------------------------------------

<br>
1. https://parkinsonsroadmap.org/research-network/pd-functional-genomics/
2. https://pubmed.ncbi.nlm.nih.gov/29289683/
3. https://discovery.ucl.ac.uk/id/eprint/10119171/
4. https://www.ucl.ac.uk/school-life-medical-sciences/about-slms/office-vice-provost-health/academic-careers-office/career-schemes/clinician-coders
5. https://github.com/ClinicianCoders/ClinicianCoders
6. https://publons.com/researcher/3017104/regina-hertfelder-reynolds/peer-review/
7. https://pubmed.ncbi.nlm.nih.gov/37117178/
8. https://rhreynolds.github.io/neurodegen-psych-local-corr/
9. https://pubmed.ncbi.nlm.nih.gov/34309761/
10. https://rhreynolds.github.io/LBD-seq-bulk-analyses/
11. https://pubmed.ncbi.nlm.nih.gov/31603214/
12. https://pubmed.ncbi.nlm.nih.gov/31016231/
13. https://www.alzforum.org/news/conference-coverage/expression-expression-expression-time-get-board-eqtls

